doc/api/crypto.md

```diff
@@ -60,11 +60,12 @@ const cert1 = new crypto.Certificate();
 const cert2 = crypto.Certificate();
 ```
 
-### certificate.exportChallenge(spkac)
+### certificate.exportChallenge(spkac[, encoding])
 <!-- YAML
 added: v0.11.8
 -->
 - `spkac` {string | Buffer | TypedArray | DataView}
+- `encoding` {string} (Default `utf-8`) used to encode the `spkac` into a Buffer
```

Comment:
For string literal values, single quotes should be added inside the backticks: \`'utf-8'\`